lib/Field/RepeatableSettings.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { bool, number } from 'prop-types';
+import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
+import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
+
+const RepeatableSettings = ({
+  isTextDirLTR,
+  fieldIsActive,
+  repeatPosition,
+  isLastRepeat
+}) => {
+  const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
+
+  return (
+    <div
+      className={`
+      text-neutral-primary
+      text-lg
+      leading-snug
+      flex
+      items-center
+      -my-3
+      ${isTextDirLTR ? 'ml-auto' : 'mr-auto'}
+    `}
+    >
+      <div
+        className={`
+        ${fieldIsActive ? '' : 'opacity-0'}
+        group-hover:opacity-100
+        transition-opacity
+        duration-200
+        ease-animation-curve
+        flex`}
+      >
+        {repeatPosition > 0 && (
+          <ButtonIconDanger name="trash" className="repeat-icon m-0" />
```

Comment:
Somewhat annoyingly, I can't override the margin on these components because of the way the ButtonIcon is written... adding `m-0` doesn't have any effect. I wonder if we could rearrange the way these classNames are merged in the ButtonIcon, so we can override our own styles - any ideas @altaudio ? - context is I wanted to reduce the horizontal margin between the buttons.

Reply:
I've also had an issue with classes not overriding others. I found. that re-ordering the classes did little too. I'd be interested in hearing a solution

Reply:
I and @DavidHollins6 found out why today. The class order in the CSS file is the reason. `m-0` is before the other classes so it will always lose.